### PR TITLE
WIP Download Resolutions

### PR DESCRIPTION
**Pull request by LRTNZ**
Basic additions for the download resolution stuff, that will hopefully help with getting variable download
resolutions up and running. Also changes stuff so the HD stream is the default one that will be downloaded
when using our download button.

### CODE REVIEW

File: plus.js [hunk 1]
```diff
@@ -109,13 +117,32 @@ document.arrive(".shaka-volume-bar-container", function() {
         // play/pause button
         play_button = "<button class='material-icons' id='mpp-play' aria-label='Play/Pause' title='Play/Pause'>play_arrow</button>";
         document.getElementsByClassName("shaka-current-time")[0].insertAdjacentHTML("beforebegin", play_button);
-        document.getElementById("mpp-play").addEventListener('click', function() {
-            if (vid.paused) {
-                vid.play();
+
+         // Workaround to stop the spacebar "clicking" as a generic keyevent causing a double play/pause
```
Comment (acoollevel) [inline plus.js:121]: What is this about?

Comment (LRTNZ) [reply plus.js:121]: Good Question - I had not rebased my branch, so I think git had a fit.

**Comment by LRTNZ** [comment]
Right, should have the fixes in now.

**Comment by LRTNZ** [comment]
Note there will be more changes in the future, but this is just to get the basics stuff there to encourage some thinking about it all.

**Pull request merged**